### ISSUE TITLE
Code Explorer should show References to other VBA projects, just like Project Explorer

### ISSUE THREAD

**Issue by ThunderFrame** [rubberduck-vba/Rubberduck#1593]
![refs](http://i.imgur.com/TYfQgLD.png)

The visual clue is useful in Project Explorer, but Code Explorer doesn't provide the hint.

**Comment by retailcoder** [comment]
The *Code Explorer* will do **much** better than that once we get #950 references dialog redone. :wink:

**Comment by Hosch250** [comment]
Is this (#950) going to be part of the CE, or in its own window?

**Comment by retailcoder** [comment]
@Hosch250 we'll make a dedicated `References` folder (the blue folder icons) under each project, and list *all* references - not just references to other VBA projects. "Find all references" will then locate all code that references anything from the selected project reference, fully qualified or not. The context menu will need an "Add/Remove references..." item, to bring up the RD references dialog.

**Comment by retailcoder** [comment]
Hmm we could also dim references that aren't used anywhere.

**Comment by ThunderFrame** [comment]
@retailcoder like #1733?

**Comment by ThunderFrame** [comment]
Need to remember that just declaring a Type is enough to be *using* a reference. eg:

```vb
'wbk is early-bound, so the Excel reference is used.
Dim wbk As Excel.Application
Set wbk = CreateObject(,"Excel.Application")
```

**Comment by comintern** [comment]
Closed via #4601
